executable-spec/src/Cardano/Ledger/Spec/STS/Common/Compact.hs

```diff
@@ -0,0 +1,50 @@
+{-# LANGUAGE DeriveAnyClass             #-}
+{-# LANGUAGE DeriveGeneric              #-}
+{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE DerivingVia                #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE OverloadedStrings          #-}
+{-# LANGUAGE FlexibleContexts           #-}
+
+
+module Cardano.Ledger.Spec.STS.Common.Compact where
```

Comment:
We don't need to copy the entire module, only use the to/from short bytestring functions.